src/app/build/build.component.ts

```diff
@@ -19,14 +20,17 @@ import { LongDateFormatKey } from 'moment';
 
 export class BuildComponent implements OnInit {
   build: Build;
+  buildlogs: BuildLog;
   revision: Revision;
   worker: BuildWorker;
   jobs: Job[];
 
+
   constructor(private route: ActivatedRoute) {}
 
   ngOnInit() {
     this.build = this.route.snapshot.data['build'];
+    this.buildlogs = this.route.snapshot.data['buildlogs'];
```

Comment:
To avoid this kind of errors at runtime, we might want to have a type check at compile time:

```
  ngOnInit() {
    this.build = this.route.snapshot.data['build'] as Build;
    this.buildlogs = this.route.snapshot.data['buildlog'] as BuildLog;
    this.revision = this.build.revision as Revision;
    this.worker = this.build.worker as BuildWorker;
    this.jobs = this.route.snapshot.data['jobs'] as Job[];
  }
```

However, this would still not catch any issues with `BuildLog`, as it's an empty interface - maybe have a single property for `message`?